rfc/multiple-go-modules.md

```diff
@@ -0,0 +1,188 @@
+| api/server       | pkg/apiserver                 | api/server                       | api/server/v0.7.0                       |
+| api/client       | pkg/shared/backendclient      | api/client                       | api/client/v0.7.0                       |
+| cli              | pkg/cli                       | cli                              | cli/v0.7.0                              |
+| cli              | pkg/shared/analyzer           | cli/pkg/analyzer                 | cli/v0.7.0                              |
+| cli              | pkg/shared/config             | cli/pkg/config                   | cli/v0.7.0                              |
+| cli              | pkg/shared/converter          | cli/pkg/converter                | cli/v0.7.0                              |
+| cli              | pkg/shared/families           | cli/pkg/families                 | cli/v0.7.0                              |
+| cli              | pkg/shared/findingkey         | cli/pkg/findingkey               | cli/v0.7.0                              |
+| cli              | pkg/shared/job_manager        | cli/pkg/jobmanager               | cli/v0.7.0                              |
+| cli              | pkg/shared/scanner            | cli/pkg/scanner                  | cli/v0.7.0                              |
+| cli              | pkg/shared/utils              | cli/pkg/utils                    | cli/v0.7.0                              |
+| orchestrator     | pkg/orchestrator              | orchestrator                     | orchestrator/v0.7.0                     |
+| uibackend        | pkg/uibackend                 | uibackend                        | uibackend/v0.7.0                        |
+| uibackend/server | pkg/uibackend/rest            | uibackend/server                 | uibackend/server/v0.7.0                 |
+| uibackend/client | pkg/shared/uibackendclient    | uibackend/client                 | uibackend/client/v0.7.0                 |
+| utils            | pkg/version                   | utils/version                    | utils/v0.7.0                            |
+| utils            | pkg/shared/command            | utils/command                    | utils/v0.7.0                            |
+| utils            | pkg/shared/fsutils            | utils/fsutils                    | utils/v0.7.0                            |
+| utils            | pkg/shared/log                | utils/log                        | utils/v0.7.0                            |
+| utils            | pkg/shared/manifest           | utils/manifest                   | utils/v0.7.0                            |
+| provider         | pkg/orchestrator/provider     | provider                         | provider/v0.7.0                         |
+| provider         | pkg/containerruntimediscovery | provider/common/runtimediscovery | provider/common/runtimediscovery/v0.7.0 |
```

Comment:
I would move the `e2e/testenv` into a separate module so we can reuse it outside the `e2e` too.

Reply:
@chrisgacsal what do you think about adding a test folder with the two modules (e2e and testenv) inside? Like
```
| Module       | Current path  | New path     | Version Tag         |
| ------------ | ------------- | ------------ | ------------------- |
| test/e2e     | e2e/          | test/e2e     | test/e2e/v0.7.0     |
| test/testenv | e2e/testenv   | test/testenv | test/testenv/v0.7.0 |
```

Reply:
added 👍